NEWS.md

```diff
@@ -4,6 +4,30 @@ Trixi.jl follows the interpretation of [semantic versioning (semver)](https://ju
 used in the Julia ecosystem. Notable changes will be documented in this file
 for human readability.
 
+## Changes when updating to v0.5 from v0.4.x
+
+#### Added
+
+#### Changed
+
+- Compile-time boolean indicators have been changed from `Val{true}`/`Val{false}`
+  to `Trixi.True`/`Trixi.False`. This affects user code only if new equations
+  with nonconservative terms are created. Change
+  `Trixi.has_nonconservative_terms(::YourEquations) = Val{true}()` to
+  `Trixi.has_nonconservative_terms(::YourEquations) = Trixi.True()`.
+
```

Comment:
```suggestion
- The (non-exported) DGSEM function `split_form_kernel!` has been renamed to `flux_differencing_kernel!`

```
I dunno if we should include it in the news, but since this has been a staple of Trixi for ~2 years, I thought it might be helpful to mention it here even though it is not part of the official API.